client/core/helpers.go

```diff
@@ -310,29 +310,38 @@ func (ord *OrderReader) StatusString() string {
 
 // SimpleRateString is the formatted match rate.
 func (ord *OrderReader) SimpleRateString() string {
+	if ord.Type == order.MarketOrderType {
+		return ord.AverageRateString()
+	}
```

Comment:
`SimpleRateString` is used exclusively in CSV-formatted files where there is an expectation of machine-readability. `AverageRateString` can add tildes and whitespace `"~ "`. :Also, backwards compatibility issues. Better leave this as `"0"` for market orders here.

Reply:
Okay.